src/routes/docs/products/functions/examples/+page.markdoc

```diff
@@ -706,11 +707,11 @@ def main(context)
             'content' => formData['content']
         }
 
-        client = Appwrite::Client.new()
-        client
-            .set_endpoint('https://cloud.appwrite.io/v1')
-            .set_project(ENV['APPWRITE_FUNCTION_PROJECT_ID'])
-            .set_key(ENV['APPWRITE_API_KEY'])
+        # Set API key
+          client = Appwrite::Client.new
```

Comment:
shouldn’t be indented. And how come the parenthesis were removed?

Reply:
I was following the Ruby code snippet examples in [Functions API reference](https://appwrite.io/docs/references/cloud/server-ruby/functions) to ensure consistency and accuracy was maintained.